docs/Community/participants.md

```diff
@@ -232,3 +232,4 @@ If you notice your a credential beeing outdated or an address missing here pleas
 | Shell                               | 0x550B078c243b0893CCA0e1Af93C5e431d7Fd080B | Participant   | |
 | pro-micron GmbH                     | 0x6C09c6561618E84052204Efdd1a5844a8EA2cAd5 | Participant   | |
 | Goudappel BV                        | 0x56289213416634216ff0BccF3671653863CE6aa1 | Participant   | |
+| Posedio.                            | 0x1E423FD6c2Ce13E34bF0dB8f0Ad5CCF372578C33 | Participant   | |
```

Comment:
question(blocking): Is it correct that it is "Posedio." with a dot? Or should it just be "Posedio"?